src/main/java/net/sf/jabref/cli/ArgumentProcessor.java

```diff
@@ -536,54 +538,65 @@ public boolean isBlank() {
 
     private static Optional<ParserResult> importFile(String argument) {
         String[] data = argument.split(",");
-        OutputPrinter printer = new SystemOutputPrinter();
 
-        try {
-            if ((data.length > 1) && !"*".equals(data[1])) {
-                System.out.println(Localization.lang("Importing") + ": " + data[0]);
-                try {
-                    Path file;
-                    if (OS.WINDOWS) {
-                        file = Paths.get(data[0]);
-                    } else {
-                        file = Paths.get(data[0].replace("~", System.getProperty("user.home")));
-                    }
-                    ParserResult result = Globals.IMPORT_FORMAT_READER.importFromFile(data[1], file);
+        String address = data[0];
+        Path file;
+        if (address.startsWith("http://") || address.startsWith("https://")) {
+            // Download web resource to temporary file
+            try {
+                file = new URLDownload(address).downloadToTemporaryFile();
+            } catch (IOException e) {
+                System.err.println(
+                        Localization.lang("Problem downloading from %1", address) +
+                        e.getLocalizedMessage());
+                return Optional.empty();
+            }
+        } else {
+            if (OS.WINDOWS) {
+                file = Paths.get(address);
+            } else {
+                file = Paths.get(address.replace("~", System.getProperty("user.home")));
```

Comment:
I didn't add this, it was already there. Since I have no linux system right now to check if it works without the special "~" handling, I prefer not to change it right now. Feel free to open a new PR.